src/main/scad/alfons-eyckmans/aracna/eyckmans.aracna.scad

```diff
@@ -0,0 +1,55 @@
+/* ====================================================================
+
+  This is a 3D model of a mechanical puzzle. It is released under
+  the following license:
+
+  Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported
+  https://creativecommons.org/licenses/by-nc-nd/3.0/
+
+  This means the model is licensed for personal, noncommercial use
+  only. Anyone may print a copy for their own use, but selling or
+  otherwise monetizing the model or print (or any derivatives) is
+  prohibited. For details, refer to the summary at the above URL.
+
+  Puzzle design (c) Alfons Eyckmans
+  3D model (c) Tom Burns
+
+==================================================================== */
+
+include <puzzlecad.scad>
+
+require_puzzlecad_version("2.1");
+
+$burr_scale = 9;
+$burr_bevel = 1;
```

Comment:
I suggest using $burr_bevel = 0.5, and then within sticks(), set $burr_outer_x_bevel = 1.75. This will ensure the interior lines of the pieces are crisp while providing a nice bevel on the ends.

Also, in the past I found that the Eyckmans zoo burrs are pretty loose-fitting and used a tighter $burr_inset than the default (cf. Save the Gorilla, Rhinoceros), with a looser inset for the "animals". If you've done a test print of Aracna and found the fit to work well, then great; if it seems a bit loose, we might consider copying the inset settings from the others.

Reply:
Outer bevels updated! This one actually comes out just right for me (maybe even a touch snug) at stock insets; this is based on the scad file I used for my store copies.